k8-cluster/on-prem/rke2/README.md

```
@@ -8,60 +8,84 @@ The following guide uses [RKE2](https://docs.rke2.io/) to set up the Kubernetes
-  * Update `hosts.ini` with proper details of all the VM's/machines.
-  * Execute Ansible command to open port on each nodes:
-    ```
-    ansible-playbook -i hosts.ini ports.yaml
-    ```
-  * Disable swap (perhaps not needed as swap is already disabled).
-    ```
-    ansible-playbook -i hosts.ini swap.yaml
-    ```
+
+#### Ports to be opened in server nodes:
+|Protocal|Port|Accesibility|Description|
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)